roles/reproducer/tasks/configure_controller.yml

```diff
@@ -300,3 +292,67 @@
       ansible.builtin.copy:
         dest: "/home/zuul/reproducer-variables.yml"
         content: "{{ _filtered_vars | to_nice_yaml }}"
+
+    - name: Get interfaces-info content
+      register: _nic_info
+      ansible.builtin.slurp:
+        src: "{{ _ctl_reproducer_basedir }}/parameters/interfaces-info.yml"
+
+    # We detected OCP cluster may have some downtime even after it's supposed
+    # to be started.
+    # It means everything was right from the libvirt_manager point of view, it
+    # could get access, configure the needed bits and move on.
+    # But when we hit this step, cluster may be unresponsive - suspecting
+    # some NetworkManager or related tool doing some "late" tasks.
+    # This task here is to ensure all of the OCPs nodes are ready to be
+    # consumed by the networking_mapper - it will connect to all of the nodes
+    # in order to gather some facts of interest.
+    - name: Wait for OCP nodes to be ready
+      delegate_to: "{{ item }}"
+      ansible.builtin.wait_for_connection:
+        sleep: 2
+        timeout: 300
+      loop: "{{ groups.ocps }}"
+
+    - name: Generate networking definition
+      vars:
+        cifmw_networking_mapper_ifaces_info: >-
+          {{ _nic_info.content | b64decode | from_yaml }}
+        cifmw_networking_mapper_network_name: >-
+          {{ _layout.vms.controller.nets.1 }}
```

Comment:
Assuming that `osp_trunk` is the second item in the nets list sounds fragile.

Reply:
it can't be first, that's a sure thing. Can't be third - or a least I don't know what kind of layout that would be.

There's no simple way. We can't enforce `osp_trunk` name (especially since I think we may use `default` if available), so there's no way to get it without fixing one thing. Or getting yet a new parameter to namely write the network name - but it's not really good either imho :(. Already have one for the "public network", and I'm unhappy with it.

Reply:
Agree that both solutions looks ugly :(  I don't have other options on top of head